package.json

```diff
@@ -100,7 +100,7 @@
   "husky": {
     "hooks": {
       "commit-msg": "commit-message-validator",
-      "pre-commit": "ng lint"
+      "pre-commit": "ng lint && ng test --watch=false"
```

Comment:
Parece que no esta funcionando muy bien.

Dañe un par de tests y en la hook solo detecta el  **lint** pero no los test

`"pre-commit": "ng lint && ng test --watch=false"
`
 hice la prueba cambiando el  orden

`"pre-commit": "ng test --watch=false && ng lint"
`y solo detecta  los test y no el lint.

como recomendación sería bueno que uses en los test  --browsers=ChromeHeadless
algo así
`"pre-commit": "ng test --watch=false --browsers=ChromeHeadless & ng lint" `

Reply:
Thank you for your comments, Sandro. It worked when I was testing the changes on my computer, maybe the problem was that I did not commit the package-lock.json file which I know now that automatically changes when changing the package.json file.

I have tested it again, and it works as expected. It does not allow you to perform the commit if a test fails or if the lint fails.